gnbsim.go

```diff
@@ -62,65 +67,59 @@ func action(c *cli.Context) error {
 	logger.AppLog.Infoln("Setting log level to:", lvl)
 	logger.SetLogLevel(lvl)
 
-	profile.InitializeAllProfiles()
+	prof.InitializeAllProfiles()
 	err := gnodeb.InitializeAllGnbs()
 	if err != nil {
 		logger.AppLog.Errorln("Failed to initialize gNodeBs:", err)
 		return err
 	}
 
-	summaryChan := make(chan common.InterfaceMessage)
-	result := "PASS"
+	go ListenAndLogSummary()
+
+	var appWaitGrp sync.WaitGroup
+	if config.Configuration.Server.Enable {
+		appWaitGrp.Add(1)
+		go func() {
+			defer appWaitGrp.Done()
+			err := httpserver.StartHttpServer()
+			if err != nil {
+				logger.AppLog.Infoln("StartHttpServer returned :", err)
+			}
+		}()
+
+		signalChannel := make(chan os.Signal, 1)
+		signal.Notify(signalChannel, os.Interrupt, syscall.SIGTERM)
+		go func() {
+			<-signalChannel
```

Comment:
What kind of signal you are expecting ?

Reply:
Ajay,  we are expecting OS SIGTERM (15)signal, which is the default signal sent when firing the "kill <pid>" command